pyInterface/package/utils/_fitTreeUtils.py

```diff
@@ -0,0 +1,24 @@
+
+
+import pyRootPwa
```

Comment:
As these import are all over the package directory, it seems python is able to handle it...